.github/DISCUSSION_TEMPLATE/proposal.yml

```diff
@@ -0,0 +1,49 @@
+name: "Proposal"
+description: "Propose a new Astro feature"
+labels: ["Proposal"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Proposal
+
+        Welcome! 👋
+
+        If you are interested in proposing a new feature or improvement to Astro, 
+        then you are in the right place! 
+
+        This template covers most of the questions you will be asked about your proposal.
+        This template is completely optional, so feel free to delete it if you'd like and 
```

Comment:
> "so feel free to delete it"

Is this possible using this "structured form" style of template? The idea/spirit behind the new roadmap is that structure is added as you go. And I'll admit, I'm a bigger fan of the "Markdown-with-comments" template vs. the "structured form" when the thing being written is more unstructured/freeform (like an early feature idea). 

It gives the author the ability to modify the template based on the context of what they're proposing. Or, just delete it with a quick `CMD-A + Delete`.

Reply:
Ah right, just copy + pasted. Yeah I'm not sure I really like the structured form versus the markdown template. It's a bit more user friendly but at the expense of the format not being consistent across stages.

Reply:
Switched this up so we're using the templates, but they only have this content prefilled